cost-analyzer/values.yaml

```diff
@@ -22,6 +22,10 @@ global:
     proxy: true # If true, the kubecost frontend will route to your grafana through its service endpoint
 #    fqdn: cost-analyzer-grafana.default.svc
 
+  # Enable only when you are using GCP Marketplace ENT listing. Learn more at https://console.cloud.google.com/marketplace/product/kubecost-public/kubecost-ent    
```

Comment:
@linhlam-kc Could you provide more comment on what the purpose of the `ubbagent` is and why it is necessary when enabling Kubecost via the GCP Marketplace?